src/TabbedBehavior.js

```diff
@@ -83,6 +83,8 @@ define(function (require) {
             tabView.$el.addClass('selected');
             var newView = new (tab.get('ContentView'))(options);
             this.view.content.show(newView);
+
+            this.view.trigger('marionette:tabs:switch', {tab: tab});
```

Comment:
are they also using a different tab mechanism on the same view? If they are, they deserve it.